### ISSUE TITLE
parametric action example 'PPO' algo does not work

### ISSUE THREAD

**Issue by AprilXiaoyanLiu** [ray-project/ray#10265]
for the example the library gives 'parametric action example', I found when having the run as default 'PPO', it failed. after printing out, it looks like when do batch prediction process, the output of model is nan, is it because the gradient explodes?

**Comment by ericl** [comment]
Can you provide a reproduction script (see the bug template)

**Comment by AprilXiaoyanLiu** [comment]
https://github.com/ray-project/ray/blob/master/rllib/examples/parametric_actions_cartpole.py   Both my script and this example has the same issue.  I run this example with policy as PPO and using torch framework. All other parameters remain the same. The bug is "RuntimeError: invalid multinomial distribution (encountering probability entry < 0)",  after further investigation, I found the model prediction results are all nan value. When using DQN as policy, it works. When using PPO, I saw this bug.
